src-vue/src/views/ThunderstoreModsView.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+    <div style="height: calc(100% - var(--fc-menu_height))">
+        <div v-if="mods.length === 0" class="fc__changelog__container">
+            <el-progress :show-text="false" :percentage="50" :indeterminate="true" />
+        </div>
+        <el-scrollbar v-else class="container">
+            <div class="card-container">
+                <!-- Search filters -->
+                <div class="filter_container">
+                    <el-input v-model="input" placeholder="Search" clearable @input="onFilterTextChange" />
+                    <!-- Message displayed when user is typing in search bar -->
+                    <div v-if="userIsTyping" class="modMessage search">
+                        Searching mods...
+                    </div>
+                </div>
+
+                <!-- Message displayed if no mod matched searched words -->
+                <div v-if="filteredMods.length === 0 && input.length !== 0 && !userIsTyping" class="modMessage">
+                    No matching mod has been found.<br/>
+                    Try another search!
+                </div>
+
+                <!-- Mod cards -->
+                <thunderstore-mod-card v-for="mod of modsList" v-bind:key="mod.name" :mod="mod" />
+            </div>
+        </el-scrollbar>
+    </div>
+</template>
+
+<script lang="ts">
+import {defineComponent} from 'vue';
+import {ThunderstoreMod} from "../utils/thunderstore/ThunderstoreMod";
+import ThunderstoreModCard from "../components/ThunderstoreModCard.vue";
```

Comment:
Shouldn't this be:

```suggestion
import { ThunderstoreModCard } from "../components/ThunderstoreModCard.vue";
```
or am I missing something? ^^"

Reply:
Yeah, you can import code in many different ways in Javascript :)
Components are not exposed the same way as JavaScript modules though.